App.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./App.css";
+
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      score: 0,
+      point: 1,
+    }
+  }
+  click = () => {
+    this.setState({score: this.state.score + this.state.point})
+  }
+  alert = () => {
+    alert("You can't afford that!")
+  }
+  scoreSpent = () => {
+    this.setState({score: this.state.score - 10})
+    this.setState({point: this.state.score + 1})
+  }
```

Comment:
```suggestion
  scoreSpent = () => {
    this.setState({score: this.state.score - 10})
    this.setState({point: this.state.point + 1})
  }
```
This one error on line 20 is the _only_ reason your tests aren't passing. Each time a user buys an upgrade for 10 points, you want `**this.state.point**` to increase by one.

 The way your code is written currently: `this.setState({point: this.state.score + 1})` is effectively increasing `this.state.point` by whatever the current score is, but you really only want to increase `this.state.point` by 1